cmd/kubelet/app/options/options.go

```diff
@@ -265,6 +269,7 @@ func applyLegacyDefaults(kc *kubeletconfig.KubeletConfiguration) {
 type KubeletServer struct {
 	KubeletFlags
 	kubeletconfig.KubeletConfiguration
+	KubeletInstanceConfiguration kubeletconfig.KubeletInstanceConfiguration
```

Comment:
Remember to call the validation function for the instance config in ValidateKubeletServer, below (we should at least add the fn even if there's nothing in it yet).